src/CameraControls.ts

```diff
@@ -1117,6 +1137,60 @@ export class CameraControls extends EventDispatcher {
 
 		};
 
+		const lockPointer = (): void => {
+
+			if ( ! this._enabled || ! this._domElement ) return;
+
+			this.cancel();
+
+			// Element.requestPointerLock is allowed to happen without any pointer active - create a faux one for compatibility with controls
+			const pointer: PointerInput = {
+				pointerId: 1,
```

Comment:
maybe we should use a unique id for pointer-lock exclusively.
```suggestion
				pointerId: - 1,
```

Reply:
Pointer id of the pointer locked pointer is `1` (since there's only one) - not sure if it's modifiable 👀

Reply:
Ah, okay I checked other pointer entries, and it seems so.
let me mark this comment as resolved.